src/pages/bike.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import Img from "gatsby-image";
+import { useStaticQuery, graphql } from "gatsby";
+
+import Layout from "../components/layout";
+import SEO from "../components/seo";
+import InfoBlock from "../components/info_block";
+
+import SubpageHeading from "../components/subpage_heading";
+import styled from "styled-components";
+
+import MHP_purple_crosshair from "../images/crosshair_purple.svg";
+
+const CrosshairLayout = styled(Layout)`
+  background: repeat center/50px url(${MHP_purple_crosshair});
+`;
+
+const StatsSubheading = styled.p`
+  text-transform: uppercase;
+  padding: 0px;
+  margin: 0px;
+`;
+
+const StatsInfo = styled.h2`
+  background: var(--MHP-purple);
+  color: var(--MHP-white);
+`;
+
+const WhiteDiv = styled.div`
+  background: var(--MHP-white);
```

Comment:
In figma, all the pages have a black background - are we going for black or white?